lib/queue_bot/command.ex

```
@@ -135,22 +154,23 @@ defmodule QueueBot.Command do
   defp parse_command(%{"payload" => payload}) do
     parsed_payload = payload |> Poison.decode!
     channel_id = parsed_payload["channel"]["id"]
+    user = parsed_payload["user"]["name"]
```

What would you think about `get_in/2`?
```elixir
user = get_in(parsed_payload, ["user", "name"])
```
That won't throw an exception if `parsed_payload["user"]` isn't a map. On the other hand, you might want to throw an exception in that case.